composedtaskrunner-task-app-dependencies/pom.xml

```diff
@@ -11,13 +11,13 @@
 	<parent>
 		<artifactId>spring-cloud-dependencies-parent</artifactId>
 		<groupId>org.springframework.cloud</groupId>
-		<version>2.0.2.RELEASE</version>
+		<version>2.1.0.BUILD-SNAPSHOT</version>
 		<relativePath />
 	</parent>
 
 	<properties>
-		<spring-cloud-task.version>2.0.0.RELEASE</spring-cloud-task.version>
-		<spring-cloud-dataflow.version>1.5.0.RELEASE</spring-cloud-dataflow.version>
+		<spring-cloud-task.version>2.1.0.BUILD-SNAPSHOT</spring-cloud-task.version>
```

Comment:
Given SCDF 1.7.0.RELEASE uses spring cloud task `1.3.0.RELEASE` should this also be `1.3.0.RELEASE`?

Reply:
This is for the 2.x release of CTR so for these dependencies we need to stay on the 2.x. line.  If we backport to the 1.3.x of CTR then yes we'd use those dependencies.

Reply:
ok, as long as we have tests running against SCDF 1.7.0.RELEASE and they all look good this would be fine :-)